build/dependencies.props

```diff
@@ -3,9 +3,9 @@
     <HtmlAgilityPackPackageVersion>1.5.1</HtmlAgilityPackPackageVersion>
     <MicroBuildCorePackageVersion>0.3.0</MicroBuildCorePackageVersion>
     <MicrosoftDotNetPlatformAbstractionsVersion>2.0.0</MicrosoftDotNetPlatformAbstractionsVersion>
-    <MicrosoftDotNetSignToolPackageVersion>1.0.0-beta.19119.1</MicrosoftDotNetSignToolPackageVersion>
-    <MicrosoftDotNetSignCheckPackageVersion>1.0.0-beta.19119.1</MicrosoftDotNetSignCheckPackageVersion>
-    <MicrosoftNETFrameworkReferenceAssembliesPackageVersion>1.0.0-alpha-5</MicrosoftNETFrameworkReferenceAssembliesPackageVersion>
+    <MicrosoftDotNetSignToolPackageVersion>1.0.0-beta.19167.10</MicrosoftDotNetSignToolPackageVersion>
+    <MicrosoftDotNetSignCheckPackageVersion>1.0.0-beta.19167.10</MicrosoftDotNetSignCheckPackageVersion>
+    <MicrosoftNETFrameworkReferenceAssembliesPackageVersion>1.0.0-alpha-004</MicrosoftNETFrameworkReferenceAssembliesPackageVersion>
```

Comment:
What I meant was, isn't this a downgrade?

Reply:
Ah, I missed that and should have left this line unchanged. Will probably require manual intervention i.e. nothing in GitHub to fix my mistake.

Reply:
What is this package anyway? I assume the downgrade is probably not desired but I actually have very little idea what's the difference between alpha-5 and alpha-004. In fact why are we using an alpha version anyway?

Reply:
It's the package containing the reference assemblies for .NET Framework. I _upgraded_ this in release/2.2. Should leave it alone here.

If you like, reassign to me and I'll do the merge locally then push to the bot's branch.